cores/arduino/main.cpp

```diff
@@ -45,6 +45,9 @@ int main( void )
 
 	for (;;)
 	{
+		// Define by Ethernet library. It is defined as __weak.
+		stm32_eth_scheduler();
```

Comment:
Think about add registration callback mechanism during instance init. This way it was more generic and usable for further lib